src/main/java/Cart.java

```diff
@@ -0,0 +1,8 @@
+public abstract class Cart {
+
+    public abstract String getShoppingContent();
+
+    public abstract String getNonExistingContent();
```

Comment:
makes sense to have some methods with arguments for future examples...

also, methods can be named like 'methodWithResultWithoutArguments', 'voidMethodWithoutArguments' etc. -> e.g. total 4 cases :)